chain/find.go

```diff
@@ -26,7 +28,7 @@ var FullTaskSet mapset.Set
 
 func init() {
 	FullTaskSet = mapset.NewSet()
-	for _, t := range AllTasks {
+	for _, t := range indexer.AllTasks {
```

Comment:
Not obvious where `indexer` comes from and should be more obvious where that is. I'd also like assurance (verbal or explicitly in code) that this is the only mutation and won't change accidentally. (Concerned about future changes potentially triggering this footgun.)

Reply:
indexer is a package, `AllTasks` is a variable from the package. This is a mechanical change since the indexer was move into a new package.